docs/lang/helloworld.md

```diff
@@ -1,76 +1,76 @@
 disqus:
 
-## 环境配置
+## Environment configuration
 
-工欲善其事，必先利其器。
+"The expectations of life depend upon diligence; the mechanic that would perfect his work must first sharpen his tools." - Confucius
 
-### 集成开发环境
+### Integrated development environment
 
-IDE 操作较为简单，一般入门玩家会选用 IDE 来编写代码。在竞赛中最常见的是 [Dev-C++](./editor/devcpp.md) （如果考试环境是 Windows 系统，一般也会提供这一 IDE）。
+IDE operation is relatively easy, which is generally the tool  beginners would choose to write code. The most common one in OI is [Dev-C++](./editor/devcpp.md) (if the testing environment is the Windows system, this IDE is also provided).
 
-### 编译器
+### Compilers
 
 #### Windows
 
-推荐使用 GNU 编译器。需要去 [MinGW Distro](https://nuwen.net/mingw.html) 下载 MinGW 并安装。此外 Windows 下也可以选择 [Microsoft Visual C++ 编译器](https://docs.microsoft.com/en-us/cpp/build/projects-and-build-systems-cpp) ，需要去 [Visual Studio 页面](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) 下载安装。
+The GNU compiler is recommended. You need to go to [MinGW Distro](https://nuwen.net/mingw.html) to download and install MinGW. You can also choose [Microsoft Visual C++ Compiler](https://docs.microsoft.com/en-us/cpp/build/projects-and-build-systems-cpp) under Windows, which requires you to download and install under [Visual Studio page](https://visualstudio.microsoft.com/downloads/).
 
 #### macOS
 
-在终端中执行：
+Execute in the terminal:
 
 ```bash
 xcode-select --install
 ```
 
 #### Linux
 
-使用 `g++ -v` 来检查是否安装过 `g++` 。
+Use `g++ -v` to check if `g++` is installed.
 
-使用如下命令可以安装：
+Use the following command to install:
 
 ```bash
 sudo apt update && sudo apt install g++
 ```
 
-#### 在命令行中编译代码
+#### Compile the code using command line
 
-熟练之后也有玩家会使用更灵活的命令行来编译代码，这样就不依赖 IDE 了，而是使用自己熟悉的文本编辑器编写代码。
+After certain time, some players could use a more flexible command line to compile the code, so that they do not need to rely on the IDE. Instead, they can choose whichever the text editor they are familiar with to write the code.
```

Comment:
```suggestion
After a while, some players could use a more flexible command line to compile the code, so that they do not need to rely on the IDE. Instead, they can choose whichever the text editor they are familiar with to write the code.
```